### DISCUSSION TITLE
Time Dilated Past via two dimensions of time

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#209] [Cosmology Models]
In thread #160 I proposed the "Time Dilated Past hypothesis" which says that time ran slower in the past, which is why ancient light is redshifted: compared to the present clock rate the old light appears to have a lower frequency than new light created by the same process.

This is a vaguely similar idea, but it involves 2 dimensions of time.

First, suppose there is 1d of space, and 1d of time.

$$ds^2 = -c^2 dt^2 + dx^2$$

An object at rest moves through time at 1 second per second. 

Now suppose there is a 2nd dimension of time, but objects move through this dimension of time much slower. Say, 1 second along this new dimension ($\tau$) per 1 million years in the standard time dimension ($t$). Something along these lines:

$$ds^2 = -c^2 dt^2 - k^2 d\tau^2 + dx^2$$

Where k is the speed of light divided by a 1 billion or something like that.

Now in these 3 dimensions (one space, two for time), over millions of years, the contribution of the 2nd dimension should be apparent. The result is cosmic time dilation and cosmic redshift. 

Thoughts? 

Edit: On second thought either $t$ or $\tau$ need a coefficient that is a function of the other one.

**Comment by RedshiftDrift** [comment]
John Hartnett has proposed a second dimension of time, where the universe is 13.8 Gyr old in one, and 6000 years old in the other time dimension.
See [https://www.researchgate.net/publication/237767334_A_5D_spherically_symmetric_expanding_universe_is_young](https://www.researchgate.net/publication/237767334_A_5D_spherically_symmetric_expanding_universe_is_young), maybe those equations look like your idea?

He called these "physical time" and "biblical time".  A few years ago John told me he had abandoned the idea of two time dimensions (but not his faith in the bible...)
